pkg/actions/service.go

```diff
@@ -37,7 +36,8 @@ var (
 )
 
 type KVService struct {
```

Comment:
"Service"?

Reply:
It's the interface, changed the name to something else.